src/jit/compiler.cpp

```diff
@@ -4428,11 +4428,12 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, CORJIT_F
     // Stash the current estimate of the function's size if necessary.
     if (verbose)
     {
-        compSizeEstimate = 0;
+        compSizeEstimate  = 0;
         compCycleEstimate = 0;
         for (BasicBlock* block = fgFirstBB; block != nullptr; block = block->bbNext)
         {
-            for (GenTreeStmt* statement = block->firstStmt(); statement != nullptr; statement = statement->getNextStmt())
+            for (GenTreeStmt* statement = block->firstStmt(); statement != nullptr;
+                 statement              = statement->getNextStmt())
             {
```

Comment:
OK, I'll rename to stmt for now - and leave to others the broader question ;-)

---
In reply to: [77263018](https://github.com/dotnet/coreclr/pull/7027#discussion_r77263018) [](ancestors = 77263018)